internal/server/storage/drivers/driver_zfs.go

```diff
@@ -25,6 +25,12 @@ import (
 	"github.com/lxc/incus/v6/shared/validate"
 )
 
+var zfsSupportedPoolTypes = []string{
```

Comment:
Probably also add `stripe` as a type and make that the default in parseSource?

That should be cleaner to handle in the code than having to check for an empty value every time.